ttmar_starter.html

```diff
@@ -54,12 +54,41 @@
   <p class="second_chorus" style="display:none">Go on take the money and run</p>
   <p class="second_chorus" style="display:none">Go on take the money and run</p>
   <p class="second_chorus" style="display:none">Go on take the money and run</p>
-  <p class="second_chorus" style="display:none">Go on take the money and run</p>
+  <p class="second_chorus" style="display:none">Go on take the money and run
 
 
 
 
 </body>
 <script>
+var firstEvent = document.queryselector("#fist_paragraph")
+firstEvent.addEventListner("click", function
+()){document.body.style.background = "#ccc"}
```

Comment:
This code has formatting issues, as well as a syntax issue that prevent it form working as I'd expect.

Reformatted, the code would look like this:

```js
firstEvent.addEventListner("click", function() )
{
  document.body.style.background = "#ccc"
}
```

It's subtle, but notice the difference in where the close parenthesis is in the first line, and how it has to be moved *after* the curly brace.

```js
firstEvent.addEventListner("click", function() {
    document.body.style.background = "#ccc"
});
```